File: 2006-count-number-of-pairs-with-absolute-difference-k/2006-count-number-of-pairs-with-absolute-difference-k.java
class Solution {
    public int countKDifference(int[] nums, int k) {
        int c = 0;
        for(int i = 0; i < nums.length-1; i++){
            for(int j = i+1; j < nums.length; j++){
                if(Math.abs(nums[i] - nums[j]) == k)
                    c++;
            }
        }
        return c;
    }
}

File: 2006-count-number-of-pairs-with-absolute-difference-k/2006-count-number-of-pairs-with-absolute-difference-k.swift
class Solution {
    func countKDifference(_ nums: [Int], _ k: Int) -> Int {
        var cnt = 0
        var i = 0
        
        while i < nums.count-1 {
            var j = i + 1 
            while j < nums.count {
                if abs(nums[i] - nums[j]) == k { cnt += 1 }        
                j += 1
            }
            i += 1
        }
        
        return cnt
    }
}